numpy-stubs/__init__.pyi

```diff
@@ -402,14 +402,15 @@ class number(generic):
 class bool_(_real_generic): ...
 class object_(generic): ...
 
-class datetime64(_real_generic):
+class datetime64:
     @overload
     def __init__(
         self, _data: Union[datetime64, str, dt.datetime] = ..., _format: str = ...
     ) -> None: ...
     @overload
     def __init__(self, _data: int, _format: str) -> None: ...
-    def __sub__(self, other: datetime64) -> timedelta64: ...
+    def __add__(self, other: Union[timedelta64, int]) -> datetime64: ...
```

Comment:
`int`?!?!

I can see that it works, but why...?

Reply:
You're going to love this! It adds/subtracts `datetime_variable.dtype.kind`, leading to some fun things like:
```
In[22]:  np.datetime64('2001') == np.datetime64('2001-01-01')
Out[22]: True
In[23]:  (np.datetime64('2001') + 1) == (np.datetime64('2001-01-01') + 1)
Out[23]: False
```

Reply:
:man_facepalming: